src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/syntaxes/aspnetcorerazor.tmLanguage.yml

```diff
@@ -24,4 +27,117 @@ repository:
       - include: 'source.cs#expression'
     end: '\)'
     endCaptures:
-      0: { name: 'keyword.control.cshtml' }
\ No newline at end of file
+      0: { name: 'keyword.control.cshtml' }
+
+  # ----------  Implicit Expression ------------
+
+  implicit-expression:
+    name: 'meta.expression.implicit.cshtml'
+    begin: '(@)'
+    beginCaptures:
+      1: { patterns: [ include: '#transition' ] }
+    patterns:
+      - include: '#await-prefix'
+      - include: '#implicit-expression-body'
+    end: '(?=[\s<])'
+
+  implicit-expression-body:
+    patterns:
+      - include: '#implicit-expression-invocation-start'
+      - include: '#implicit-expression-accessor-start'
+    end: '(?=[\s<])'
+
+  # CSharp colors method invocations differently than accessors
+  # This captures situations where you do @SomeMethod()
+  implicit-expression-invocation-start:
+    begin: '([_[:alpha:]][_[:alnum:]]*)(?=\()'
+    beginCaptures:
+      1: { name: 'entity.name.function.cs' }
+    patterns:
+      - include: '#implicit-expression-continuation'
+    end: '(?=[\s<])'
+
+  # CSharp colors property accessors differently than methods
+  # This captures situations where you do @SomeMethod()
```

Comment:
This comment is the same as the one for `implicit-expression-invocation-start`, shouldn't this be `This captures situations where you do @SomeProperty`?